pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -324,3 +329,19 @@ type KogitoAppList struct {
 func init() {
 	SchemeBuilder.Register(&KogitoApp{}, &KogitoAppList{})
 }
+
+//Artifact contains final generated project artifact
+type Artifact struct {
```

Comment:
The attributes of this structure need to be annotated as well. Sorry if my last comment wasn't clear.

Reply:
done

Reply:
@vaibhavjainwiz the attributes of the structure is `GroupId`, `ArtifactId` and `Version`.